index.html

```diff
@@ -1,192 +1,480 @@
+                  alt="JS Logo"
+                  width="100px"
+                  height="100px"
+                  src="images/js.png"
+                />
+                <p class="js poppins">JavaScript</p>
+              </div>
+            </li>
+            <li class="li-sp">
+              <div class="row1">
+                <img
+                  class="html5-img"
+                  alt="HTML5 Logo"
+                  width="100px"
+                  height="101px"
+                  src="images/html5.png"
+                />
+                <p class="html5 poppins">HTML</p>
+              </div>
+            </li>
+            <li class="li-sp">
+              <div class="row1">
+                <img
+                  class="css3-img"
+                  alt="CSS3 Logo"
+                  width="100px"
+                  height="101px"
+                  src="images/css3.png"
+                />
+                <p class="css3 poppins">CSS</p>
+              </div>
+            </li>
+            <!-- </div> -->
+          </ul>
+          <hr />
+        </li>
+        <li>
+          <div class="frameworks">
+            <p class="frame poppins">Frameworks</p>
+            <button name="button-arrow2" type="button">
+              <img
+                class="arrow-right"
+                alt=""
+                width="512px"
+                height="512px"
+                src="images/right-arrow.png"
+              />
+            </button>
+          </div>
+          <hr />
+        </li>
+        <li>
+          <div class="skills">
+            <p class="skill poppins">Skills</p>
+            <button name="button-arrow3" type="button">
+              <img
+                class="arrow-right"
+                alt=""
+                width="512px"
+                height="512px"
+                src="images/right-arrow.png"
+              />
+            </button>
+          </div>
+          <hr />
+        </li>
+      </ul>
+    </section>
+
+    <footer class="footer">
+      <section class="contact">
+        <form id="form" action="https://formspree.io/f/mjvdqaka" method="post" class="form1">
+          <h2 class="heading2">Contact me</h2>
+          <p class="paragraph2 poppins">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding.<br>I’d love
+            to help with it!
+          </p>
+          <div class="box2">
+            <input id="name" name="name" type="text" class="poppins" placeholder="Lorem Ipsum" maxlength="30" required>
+          </div>
+          <div class="box2">
+            <input id="email" name="email" type="email" class="poppins" placeholder="loremipsum@gmail.com" required>
+          </div>
+          <div class="box3">
+            <textarea id="message" name="message" class="poppins" rows="5" cols="43" placeholder="Write your message here..." maxlength="500" required></textarea>
+          </div>
```

Comment:
Kindly, I suggest you remove all this `div` tags in replace them by `ul` and `li` to group them.

It seems you forgot to close the `input` tags element, kindly, I suggest you close the `input` as follow: `<input ... />` for good semantics and best practices. 😄